tests/robot-tests/tests/admin_and_public/bau/release_and_publication_role_permissions.robot

```diff
@@ -0,0 +1,358 @@
+
+Navigate to manage users page as bau1
+    [Tags]  HappyPath
+    user goes to url  %{ADMIN_URL}/administration/users
+    user checks table column heading contains  1  1  Name
+    user checks table column heading contains  1  2  Email
+    user checks table column heading contains  1  3  Role
+    user checks table column heading contains  1  4  Actions
+
+Assert that test users are present in table
+    [Tags]  HappyPath
+    user checks results table row heading contains  1  1  Analyst1 User1
+    user checks results table row heading contains  2  1  Analyst2 User2
+    user checks results table row heading contains  3  1  Analyst3 User3
+    user checks results table row heading contains  4  1  Bau1 User1
+    user checks results table row heading contains  5  1  Bau2 User2
+
+
+    user checks results table cell contains  1  1  	ees-analyst1@education.gov.uk
+    user checks results table cell contains  1  2  	Analyst
+    user checks results table cell contains  1  3  	Manage
+
+    user checks results table cell contains  2  1  	ees-analyst2@education.gov.uk
+    user checks results table cell contains  2  2  	Analyst
+    user checks results table cell contains  2  3  	Manage
+
+    user checks results table cell contains  3  1  	ees-analyst3@education.gov.uk
+    user checks results table cell contains  3  2  	Analyst
+    user checks results table cell contains  3  3  	Manage
+
+    user checks results table cell contains  4  1  	ees-bau1@education.gov.uk
+    user checks results table cell contains  4  2  	BAU User
+
+    user checks results table cell contains  4  3  	Manage
+
+    user checks results table cell contains  5  1  	ees-bau2@education.gov.uk
+    user checks results table cell contains  5  2  	BAU User
+    user checks results table cell contains  5  3  	Manage
+
+Give Analyst1 User1 publication owner access
+    [Tags]  HappyPath
+    # user waits until h1 is visible  Users  60
+    user clicks link  Manage  xpath://td[text()="ees-analyst1@education.gov.uk"]/..
+    user waits until page does not contain loading spinner
+    # user waits until h1 is visible  Analyst1 User1  60
+    # stale element exception if you don't wait until it's enabled    
+    user waits until button is enabled  Add publication access
+
+    user scrolls to element  css:[name="selectedPublicationId"]
+    
+    user waits until element is enabled  css:[name="selectedPublicationId"]
+    user selects from list by label  css:[name="selectedPublicationId"]  ${PUBLICATION_NAME}
+
+    user waits until element is enabled  css:[name="selectedPublicationRole"]
+    user selects from list by label  css:[name="selectedPublicationRole"]  Owner
+    
+    user clicks button  Add publication access
+    
+Sign in as Analyst1 User1 (publication owner) & navigate to publication
+    [Tags]  HappyPath
+    user signs in as analyst1
+    user waits until page does not contain loading spinner
```

Comment:
Looks like the tests are currently failing on this line - looks like the loading spinner remains until the user has scrolled down.

All that said, I think you should be able to remove this wait